components/talksPage/talksPage.module.scss

```diff
@@ -0,0 +1,76 @@
+$itemGrow: 1.2;
+$duration: 250ms;
+
+.descriptionContainer {
+  display: flex;
+  gap: 20px;
+  flex-direction: column;
+  direction: ltr;
+  width: 80%;
+  margin: 0 auto;
+}
+
+.title {
+  font-weight: 800;
+  font-size: 36px;
+  color: #39475d;
+  line-height: 47px;
+}
+
+.wrapper {
+  display: grid;
+  grid-template-columns: repeat(4, 100%);
+  overflow: hidden;
+  height: 200px;
+  scroll-behavior: smooth;
+
+  .section {
```

Comment:
do we need the nested class or can it be outside the wrapper thanks to css modules?